src/net/sf/memoranda/ui/App.java

```diff
@@ -13,6 +13,15 @@
 import net.sf.memoranda.EventsScheduler;
 import net.sf.memoranda.util.Configuration;
 
+// ajcallos 2/16/2016
```

Comment:
To avoid cluttering up the code, it'd probably be good to delete all comments of this sort.